storage-proofs-porep/Cargo.toml

```diff
@@ -54,11 +54,12 @@ pretty_env_logger = "0.4.0"
 filecoin-hashers = { path = "../filecoin-hashers", version = "^3.0.0", default-features = false, features = ["poseidon", "sha256", "blake2s"]}
 
 [features]
-default = ["pairing", "gpu"]
+default = ["pairing", "gpu", "cpu-detection"]
 gpu = ["storage-proofs-core/gpu", "filecoin-hashers/gpu", "neptune/opencl", "bellperson/gpu", "fr32/gpu"]
 pairing = ["storage-proofs-core/pairing", "bellperson/pairing", "neptune/pairing", "filecoin-hashers/pairing", "fr32/pairing"]
 blst = ["storage-proofs-core/blst", "bellperson/blst", "neptune/blst", "filecoin-hashers/blst", "fr32/blst"]
 single-threaded = []
+cpu-detection = ["hwloc"]
```

Comment:
I wonder if this is the right name, maybe this should rather be sth like "multicore-sdr"?

Reply:
also needs some docs in the readme